### PR TITLE
VACMS-19805: Adds new statuses

### PR DESCRIPTION
**Pull request by omahane**
## Description

Relates to #19805 

## Testing done
Manually

### New statuses added
Screenshot 2024-11-15 at 06 49 20

### POST API Queue with a new status
Screenshot 2024-11-15 at 06 49 43

### Confirm Operating Status changes with Vet Center - Outstation 
- Log in as an [admin](https://pr19881-d9qxbj3ygtxam1xgohoosfw3idwml7w9.ci.cms.va.gov/)
- Create a [Vet Center - Outstation](https://pr19881-d9qxbj3ygtxam1xgohoosfw3idwml7w9.ci.cms.va.gov/node/add/vet_center_outstation)
- [x] Confirm that the Operating Status is set to "Normal services and hours"
- [x] Confirm that the following options available
  - Normal services and hours
  - Limited services and hours
  - Facility closed
  - Temporary facility closure
  - Temporary location
  - Virtual care only
  - Coming soon
  - Facility notice
- [x] Confirm that the **Details** text box does not show under the Operating status when the status is set to "Normal services and hours"
- Choose "Limited services and hours", "Facility closed", "Temporary facility closure", "Temporary location", "Virtual care only", "Coming soon", and "Facility notice" in succession.
- [x] Confirm that the **Details** text box **does** show when any of those options are chosen.
- [x] Confirm that the **Details** text box is required when any of those options are chosen.

### Add the additional statuses to facilities
- Edit the [Columbia VA Regional Benefit Office](https://pr19881-d9qxbj3ygtxam1xgohoosfw3idwml7w9.ci.cms.va.gov/node/4063/edit)
- Set the **Operating status** to "Temporary facility closure"
- Add more information to **Details**
- Save the node as Published
- Edit the [Escanaba Vet Center](https://pr19881-d9qxbj3ygtxam1xgohoosfw3idwml7w9.ci.cms.va.gov/node/3756/edit)
- Set the **Operating status** to "Temporary location"
- Add more information to **Details**
- Save the node as Published
- Edit the [Duluth Vet Center](https://pr19881-d9qxbj3ygtxam1xgohoosfw3idwml7w9.ci.cms.va.gov/node/3751/edit)
- Set the **Operating status** to "Virtual care only"
- Add more information to **Details**
- Save the node as Published
- Edit the [Newport VA Clinic](https://pr19881-d9qxbj3ygtxam1xgohoosfw3idwml7w9.ci.cms.va.gov/node/2008/edit)
- Set the **Operating status** to "Coming soon"
- Add more information to **Details**
- Save the node as Published

### Check the POST API Queue
- Go to the [POST API Queue](https://pr19881-d9qxbj3ygtxam1xgohoosfw3idwml7w9.ci.cms.va.gov/admin/config/post-api/queue).
- [x] Confirm that the "Columbia VA Regional Benefit Office" queue item includes `"operating_status":{"code":"TEMPORARY_CLOSURE"`
- [x] Confirm that the "Escanaba Vet Center" queue item includes `"operating_status":{"code":"TEMPORARY_LOCATION"`
- [x] Confirm that the "Duluth Vet Center" queue item includes `"operating_status":{"code":"VIRTUAL_CARE"`
- [x] Confirm that the "Newport VA Clinic" queue item includes `"operating_status":{"code":"COMING_SOON"`

### Check the Facilities Status Audit
- Go to the [Facilities Status Audit](https://pr19881-d9qxbj3ygtxam1xgohoosfw3idwml7w9.ci.cms.va.gov/admin/content/facilities/facility-status)
- Choose each of the operating statuses from the **Operating status** filter
- Filter for each one
- [x] Confirm that the audit view is returning the appropriate facilities

### Check VAMC System Operating Status IEF
- Edit the [Operating status | VA Boston health care](https://pr19881-d9qxbj3ygtxam1xgohoosfw3idwml7w9.ci.cms.va.gov/node/27318/edit)
- Scroll to **Facility operating statuses**
- Edit the "Jamaica Plain VA Medical Center" status
- [x] Confirm that all seven statuses are available in the **Status** dropdown
- Choose "Coming soon"
- Click "Update facility operating status"
- Save the Operating status node as Published
- Go to the [POST API Queue](https://pr19881-d9qxbj3ygtxam1xgohoosfw3idwml7w9.ci.cms.va.gov/admin/config/post-api/queue).
- Confirm that the "Jamaica Plain VA Medical Center" queue item includes `"operating_status":{"code":"COMING_SOON"`
As admin, check all of the Facilities to make sure that they all behave the same way.

### Select Team for PR review

- [ ] `CMS Team`
- [ ] `Public websites`
- [x] `Facilities`
- [ ] `User support`
- [ ] `Accelerated Publishing`

### CODE REVIEW

**Comment by thejordanwood** [comment]
@omahane Followed QA steps and this is working as expected. I did notice that there are a few variations of the detail input field. 

1. This is the one I see on Columbia and Newport, which is the one I believe we should be going with. 
Screenshot 2024-11-18 at 2 29 56 PM

2. This is what I see when creating a new Vet Center - Outstation
Screenshot 2024-11-18 at 2 30 36 PM

3. This is what I see on Escanaba and Duluth.
Screenshot 2024-11-18 at 2 32 13 PM

**Review by jv-agile6 — APPROVED** [review]
Followed QA steps to successful resolution. Reviewed yml files for defects and found none. Approval granted

**Review by jv-agile6 — APPROVED** [review]
Reviewed yml changes to label and description on field.field.node.vet_center.field_operating_status_more_info.yml. No defects in code identified. Approval granted

File: config/sync/field.field.node.vba_facility.field_operating_status_more_info.yml [hunk 1]
```diff
@@ -1,19 +1,19 @@
...
     - tmgmt_content
 third_party_settings:
   tmgmt_content:
     excluded: false
-id: node.vba_facility.field_operating_status_more_info
+id: node.vet_center.field_operating_status_more_info
 field_name: field_operating_status_more_info
 entity_type: node
-bundle: vba_facility
+bundle: vet_center
```
Comment (eselkin) [inline config/sync/field.field.node.vba_facility.field_operating_status_more_info.yml]: Curious about the vet_center settings in the vba file. Is that correct? I imagine it is, but just thought I'd double check.

Comment (omahane) [reply config/sync/field.field.node.vba_facility.field_operating_status_more_info.yml]: You're right. That was a copy/paste error. I caught it. Thanks, @eselkin

**Pull request merged**